website/docs/d/cm_offering_instance.html.markdown

```diff
@@ -1,46 +1,92 @@
 ---
-subcategory: "Catalog Management"
 layout: "ibm"
 page_title: "IBM : ibm_cm_offering_instance"
 description: |-
-  Get information about ibm_cm_offering_instance.
+  Get information about cm_offering_instance
+subcategory: "Catalog Management API"
```

Comment:
Why we are changing the subcategory from 
Catalog Management or Catalog Management API